packages/local_auth/lib/local_auth.dart

```diff
@@ -72,4 +74,40 @@ class LocalAuthentication {
     }
     return await _channel.invokeMethod('authenticateWithBiometrics', args);
   }
+
+  /// Returns true or false if biometrics are available
+  ///
+  /// Returns a [Future] bool true or false:
+  Future<bool> canCheckBiometrics() async {
+    final List<dynamic> biometrics =
+        await _channel.invokeMethod('getAvailableBiometrics');
+    return biometrics.isNotEmpty;
+  }
+
+  /// Returns a list of enrolled biometrics
+  ///
+  /// Returns a [Future] List<BiometricType> with the following possibilities:
+  /// - BiometricType.face
+  /// - BiometricType.fingerprint
+  /// - BiometricType.iris (not yet implemented)
+  Future<List<BiometricType>> getAvailableBiometrics() async {
+    final List<dynamic> result =
```

Comment:
List\<string\> instead of dynamic.

Reply:
Throws an exception, that's why I use dynamic
```
[VERBOSE-2:shell.cc(181)] Dart Error: Unhandled exception:
type 'List<dynamic>' is not a subtype of type 'List<String>'
#0      LocalAuthentication.getAvailableBiometrics (package:local_auth/local_auth.dart:94:24)
```

Reply:
That's an unfortunate artifact of the plugin system as it does not allow you to generify the methods easily. You can explicitly cast it:

```dart
  (await _channel.invokeMethod(...) as List).cast<String>();
```

Reply:
I added the cast (+added type NSString for NSMutableArray)

I omitted `as List` for the cast, as it caused this warning: 
`[dart] Avoid using `as`.`